api/models/build.js

```diff
@@ -59,26 +61,25 @@ const jobStateUpdate = (buildStatus, build, timestamp) => {
     atts.completedAt = timestamp;
   }
 
+  if (buildStatus.status === 'success') {
+    atts.url = buildUrl(build, site);
+  }
+
   if (build.state === 'queued' && buildStatus.status === 'processing') {
     atts.startedAt = timestamp;
   }
 
   return build.update(atts);
 };
 
-const completeJobSiteUpdate = (build, completedAt) => {
-  const { Site } = build.sequelize.models;
-  return Site.update(
-    { publishedAt: completedAt },
-    { where: { id: build.site } }
-  );
-};
+const completeJobSiteUpdate = (site, completedAt) => site.update({ publishedAt: completedAt });
```

Comment:
This might make more sense as a function on the Site now!

Reply:
i don't think this really requires its own function. i have removed the function and just call site.update